From 7806b702a280f06f09120786bace19cd3c5c4bf7 Mon Sep 17 00:00:00 2001
From: Marian Buschsieweke <marian.buschsieweke@posteo.net>
Date: Sat, 4 Oct 2025 12:20:39 +0200
Subject: [PATCH] include/forkserver.h: add missing include

Without `#include <fcntl.h>`, `mode_t` is undeclared (at least on musl
based systems).
---
 include/forkserver.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/forkserver.h b/include/forkserver.h
index 55cf3e0b..bf924395 100644
--- a/include/forkserver.h
+++ b/include/forkserver.h
@@ -28,8 +28,9 @@
 #ifndef __AFL_FORKSERVER_H
 #define __AFL_FORKSERVER_H
 
-#include <stdio.h>
+#include <fcntl.h> /*< provides mode_t */
 #include <stdbool.h>
+#include <stdio.h>
 #include <string.h>
 #include <unistd.h>
 
-- 
2.51.0

